css-two-column-layout/styles.css

```diff
@@ -0,0 +1,76 @@
+body {
+  font-family: Karla, sans-serif;
+  display: grid;
+  color: #4a4a4a;
+}
+
+.container {
+  max-width: 1000px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.content {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  flex-basis: auto;
+  padding: 20px;
+  flex-wrap: wrap;
+}
```

Comment:
You have no styling for your row class, you are still using this to address what the row class is meant to address.